### DISCUSSION TITLE
What is the new `data-popper-placement`?

### DISCUSSION THREAD

**Discussion by tu4mo** [floating-ui/floating-ui#1487] [Q&A] [answered]
How to style the arrow differently (i.e. rotate) based on current placement of Floating UI? Popper used to have data-attributes for this.

**Comment by atomiks** [comment, marked as answer]
The resultant placement gets returned from `computePosition()`, so you could set an attribute just like in Popper.

https://floating-ui.com/docs/computePosition#return-value
